packages/kit/src/views/Onboardingv2/pages/ConnectYourDevice.tsx

```
@@ -983,18 +1006,26 @@ function BluetoothConnectionIndicator({
 
   const handleOpenPrivacySettings = useCallback(() => {
     void globalThis.desktopApiProxy.bluetooth.openPrivacySettings();
   }, []);
 
-  const { checkBluetoothStatus } = useDesktopBluetoothStatusPolling(
+  const { checkBluetoothStatus, setIsConnecting: setBluetoothConnecting } =
+    useDesktopBluetoothStatusPolling(
     tabValue,
     setBluetoothStatus,
-  );
+    );
```

ESLint

[error] Replace `⏎····tabValue,⏎····setBluetoothStatus,⏎····` with `tabValue,·setBluetoothStatus`

(prettier/prettier)